### DISCUSSION TITLE
`sprite build` to accept object json

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#22143] [Feature requests] [answered]
It would be super useful if sprite build would identify and use the images array if the top level json is an object.

**Comment by Gymnasiast** [comment]
You mean in a way similar to gxc?

**Reply by spacek531** [reply]
I don't know what features gxc has, so I can't say.

**Reply by Gymnasiast** [reply]
gxc is what compiles the images of a parkobj into a g1-like structure. We use it for the object builds. It sounds almost exactly like what you described.

**Reply by spacek531** [reply]
I perhaps should have outlined the use case... I want to be able to use an object json in `sprite build` without the chore of extracting the images table and saving as a separate fiel.

**Reply by Gymnasiast** [reply]
I don't understand. The whole point of `sprite build` is to build a standalone, g1-like file.

**Comment by spacek531** [comment, marked as answer]
This is implemented in #23640
